edx_proctoring/backends/rest.py

```diff
@@ -122,6 +124,13 @@ def get_javascript(self):
                 u'Webpack stats file corresponding to WebWorkers not found: {}'
                 .format(str(err))
             )
+
+        # if the Javascript URL is not an absolute URL (i.e. doesn't have a scheme), prepend
+        # the LMS Root URL to it, if it is defined, to make it an absolute URL
+        if not js_url.startswith('http') or not js_url.startswith('https'):
```

Comment:
you don't need the https -- `startswith('http')` is close enough.

Reply:
lol, oops